src/destinations/slack.ts

```diff
@@ -61,9 +62,12 @@ export const validateSlackWebhookUrl = (url: string): boolean => {
 
 export const sendAlertsToSlack = async (
   webhookUrl: string,
+  channel: string,
```

Comment:
Channel should be optional?

```suggestion
  channel?: string,
```